dash/_callback_context.py

```diff
@@ -10,11 +10,11 @@ def assert_context(*args, **kwargs):
         if not flask.has_request_context():
             raise exceptions.MissingCallbackContextException(
                 (
-                    'dash.callback_context.{} '
-                    'is only available from a callback!'
-                ).format(getattr(func, '__name__'))
+                    "dash.callback_context.{} " "is only available from a callback!"
```

Comment:
```suggestion
                    "dash.callback_context.{} is only available from a callback!"
```
Every once in a while Black does something very silly looking...